index.html

```diff
@@ -596,7 +608,8 @@ <h4>parseBaseProofValue</h4>
 disclosure base proof value. The required input is a proof value
 (<var>proofValue</var>). A single object, <em>parsed base proof</em>, containing
 five elements, using the names "bbsSignature", "bbsHeader", "publicKey",
```

Comment:
```suggestion
three to five elements, using the names "bbsSignature", "bbsHeader", "publicKey",
```

Reply:
I think this should be five (no pseudonym feature) or seven (`pid` and `signer_blind` present to signal pseudonym use, but possibly empty).